proposals/0000-language-editions.rst

```diff
@@ -0,0 +1,1144 @@
+Fortified Language Editions
+===========================
+
+.. author:: Richard Eisenberg (with some contributions by Simon Peyton Jones)
+.. date-accepted::
+.. ticket-url::
+.. implemented::
+.. highlight:: haskell
+.. header:: This proposal is `discussed at this pull request <https://github.com/ghc-proposals/ghc-proposals/pull/636>`_.
+.. sectnum::
+.. contents::
+
+.. _manifesto: https://github.com/ghc-proposals/ghc-proposals/pull/628
+
+I propose to lean more heavily on *language editions*. This proposal describes
+how a language edition can control more than just extensions (importantly, one
+can control *warnings* as well), and proposes several concrete language editions.
+One of the proposals is for ``Stable2024``, which will continue to be supported,
+without change to the set of user programs accepted, for at least 3 years.
```

Comment:
Should this say "6 years"? That's what the remainder of the document seems to suggest.

```suggestion
One of the proposals is for ``Stable2024``, which will continue to be supported,
without change to the set of user programs accepted, for at least 6 years.
```